migrations/20240109071854-add-lastPasswordChangedAt-to-users.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
+
+const tableName = 'users';
+const newColumn = 'last_password_changed_at';
+
+/** @type {import('sequelize-cli').Migration} */
+module.exports = {
+  async up(queryInterface, Sequelize) {
+    const tableDefinition = await queryInterface.describeTable(tableName);
+    //  Only add column if is not created in the table
+    if (!tableDefinition[newColumn]) {
```

Comment:
The migrations should be idempotent, not dependent on the environment. Do not worry about production, we will run a clean migration.